cppcache/src/ClientMetadataService.cpp

```diff
@@ -143,7 +143,12 @@ void ClientMetadataService::getClientPRMetadata(const char* regionFullPath) {
                                               reply.getFpaSet());
       if (m_bucketWaitTimeout > std::chrono::milliseconds::zero() &&
           reply.getNumBuckets() > 0) {
-        WriteGuard guard(m_PRbucketStatusLock);
+        std::unique_lock<std::timed_mutex> guard(m_timedBucketStatusLock,
+                                                 std::chrono::seconds(60));
```

Comment:
Random numbers need explanation. Hence using a well titled define or comment. Just having 60 there helps no one.

Reply:
Would it be ok to define constant like CLOSE_SERVICE_MUTEX_TIMEOUT for this ?